SegueManager.podspec

```diff
@@ -24,7 +24,14 @@ Pod::Spec.new do |s|
     ss.ios.deployment_target = '8.0'
     ss.source_files = "src/ios/SegueManager+Rswift.swift"
     ss.dependency "SegueManager/iOS"
-    ss.dependency "R.swift.Library"
+    ss.dependency "R.swift.Library", "~> 1.0.0.beta"
```

Comment:
Oh, I think I should revert this change. Only editted this so it would install the library beta. Should not be released. :)